README.md

```diff
@@ -20,60 +12,56 @@ _______
 
 # bbnavi: Documentation
 
-NOTE: This README is also available in German. Thank you for understanding that the German version might not always be up-to-date with the English one.
+NOTE: This README is also available in English. Thank you for understanding that the English version might not always be up-to-date with the German one.
```

Comment:
We can remove that note, given that the entire (and only) readme is in English, right? 😄